membrane_curvature/base.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+from membrane_curvature.lib.mods import derive_surface, mean_curvature, gaussian_curvature
+import numpy as np
+
+
+def __init__(self, universe, select,
+             x_bins=100, y_bins=100,
+             x_range=None,
+             y_range=None,
+             pbc=True):
+    self.universe = universe
```

Comment:
```suggestion
def __init__(self, universe, select,
             x_bins=100, y_bins=100,
             x_range=None,
             y_range=None,
             pbc=True, **kwargs):
    super().__init__(universe.trajectory, **kwargs)
    self.universe = universe
```

Calling the parent class `__init__` makes sure the base class instantiation is done as well!